mods/ts/rules/critters.yaml

```diff
@@ -36,6 +36,17 @@ DOGGIE:
 	HitShape:
 		Type: Circle
 			Radius: 213
+	GrantConditionOnTerrain:
+		Condition: hidingplace
+		TerrainTypes: Tiberium, BlueTiberium
+	GrantConditionOnMovement:
+		Condition: moving
+	WithIdleOverlay:
```

Comment:
`WithSpriteBody` should work as well and would be a bit more fitting.

Reply:
You are right.